provision/minikube/jaeger/deployment-patch.yaml

```diff
@@ -0,0 +1,20 @@
+# can't configure the storage of traces for badger via the helm chart
+spec:
+  template:
+    spec:
+      containers:
+        - name: jaeger
+          env:
+            - name: SPAN_STORAGE_TYPE
+              value: badger
+            - name: BADGER_SPAN_STORE_TTL
+              value: 1440h0m0s # 1440 hours = 60 days
```

Comment:
even this value, along with other retention period values, if we could create a logic to control them from one place, it would be great. If not, alternatively, it would be nice to point them out in the docs, till the point, we have some central way of controlling this value